student-bios/bibhasingh.md

```diff
@@ -1,4 +1,4 @@
-# Hello :wave:
+# Hello :wave
```

Comment:
You should add only the file with your username. Add one file at the time, please don't use `git add .`